contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,571 @@
+# Local Raw Block Consumption via Persistent Volume Source
```

Comment:
So, I noticed someone changed the title to just raw block, are we fine with this?